var/spack/repos/builtin/packages/eigen/package.py

```diff
@@ -68,6 +73,7 @@ class Eigen(CMakePackage):
         values=("Debug", "Release", "RelWithDebInfo"),
     )
 
+    depends_on("boost@1.53:", when="@master")
```

Comment:
```suggestion
```

This is [only for benchmarks and some multi-prec tests](https://gitlab.com/search?group_id=6577938&nav_source=navbar&project_id=15462818&repository_ref=master&search=boost&search_code=true) isn't it?

I don't think it should be a hard dependency, has to be optional (either on benchmarks or explicitly requesting the relevant test).

Reply:
Thanks, I changed it to type="test"